### ISSUE TITLE
Create Test Environment and Test 2.0 Build [LSRB]

### ISSUE THREAD

**Issue by patrickm02L** [cityofaustin/atd-data-tech#17161]
## Description

As part of upgrading the Living Streets application to the new 2.0 data model, a test environment needs to be created to test the new build before deploying it to production. This will allow us to:

- Test the new intake form and data mapping to ensure data is being captured correctly in the new activation cycle records
- Test connectivity between parent application records and child activation cycle records 
- Test functionality of the new data model and reporting
- Identify any bugs or issues before deploying to production
- Gather a list of tasks for deploying the new build to production

## Acceptance Criteria
- A test Knack environment has been created with a duplicate of the current Living Streets data
- The new 2.0 data model and forms have been built out in the test environment  
- Sample application records have been converted to the new activation cycle records
- Basic functionality and reporting is working as expected with the new data model

## Note

Once testing is complete and any bugs or issues have been resolved, the plan is to deploy the new 2.0 build from the test environment over to the live production environment. A date will be set for this go-live and the Living Streets team will be notified in advance of any changes or potential disruptions at that time.

**Comment by KaroEngstrom** [comment]
[heart]         Engstrom, Karo reacted to your message:
________________________________

**Comment by KaroEngstrom** [comment]
## Make Note - To Do Prior to Go-Live Window 
- [ ]  Delete unused Staging section for Applicant Portal (no unique use or build)
- [ ]  Hide breadcrumbs from the new EOI form

## To Do During the Go-Live Window
- Applicant Portal
     - **Living Streets Applicant Portal** - scene 176 - no change
          - Do we want to show an activations table for the Applicant?
- **My Application Details** - scene 215
     - [ ]

**Comment by KaroEngstrom** [comment]
## Testing Debrief with Testers - Mon 6/24/24 
- Jonathan; Manuela; Carmen
